### DISCUSSION TITLE
Limit 1 of each item being added to the basket

### DISCUSSION THREAD

**Discussion by rogerg1967** [openstore-ecommerce/OpenStore#256] [Q&A]
Is there any way to limit each product to only be added to the basket once?
We are using the system to allow customers to download templates. As such they will only ever need to add 1.
I am thinking of an if statement around the add to cart but wondered if there is a better solution.

Thanks

**Comment by DNNMonster** [comment]
In terms of the BO settings we took care of the min quantity.  Not the max qty.  https://github.com/openstore-ecommerce/OpenStore/discussions/236

Unless I'm overlooking an option, you'll have to create a solution in you're custom template as you mentioned or you could hook into one of the cart events with a plugin.
